src/markdown_generators_test.js

```diff
@@ -72,6 +78,17 @@ describe("generateMarkdownFromJson", () => {
             file["parent_uid"] === sectionUid &&
             file["file_type"] === "application/pdf"
         )
+        const expectedMedia = Object.keys(
+          singleCourseJsonData["course_embedded_media"]
+        )
+          .map(key => {
+            const embeddedMedia =
+              singleCourseJsonData["course_embedded_media"][key]
+            return embeddedMedia["parent_uid"] === sectionUid
+              ? embeddedMedia
+              : null
+          })
+          .filter(embeddedMedia => embeddedMedia)
```

Comment:
I think here you could also use `Object.values` and then just put the `embeddedMedia["parent_uid"] === sectionUid` check into your `.filter`